code/parachain/frame/composable-traits/src/xcm/memo.rs

```diff
@@ -15,13 +15,20 @@ use crate::prelude::*;
 	Debug,
 )]
 pub struct ChainInfo {
-	pub chain_id: u128,
-	pub order: u8,
+	pub chain_id: u128, // please use u32 for chain id, i asked in one of review before
+	pub order: u8, // please document what order means
```

Comment:
This looks like it should be an enum.

Reply:
Enum forced us to predefine some chain id.
But this is dynamic and will be added via sudo call just specifying chain id as a number.
So i would like to stay it like this. and do not use Enum. because it brings limitations in the process when we want to add new route.